DataFormats/StdDictionaries/src/classes_others.h

```diff
@@ -28,10 +28,6 @@ namespace DataFormats_StdDictionaries {
   std::random_access_iterator_tag randaccitertag;
   std::set<int> dummy19;
   std::set<std::basic_string<char> > dummy20;
-  __gnu_cxx::new_allocator<char> dummy21;
```

Comment:
The original PR: #57

Some notes I found in emails:
```
(13.4) We use edmClassStorageSize to iterate overall Reflex dictionaries and we find that we have ~200 errors and ~600-800 warnings coming
               from ROOT. We think it might my partially responsible for ROOT failures on ARM. We fix part of them in CMSSW_7_0_X.
```

This was related to porting to ARMv7 and some failures.

These are a snippet from some old dump:
```
[..]
## std::allocator<char>
### Errors
~~~
Error in <TStreamerInfo::Build>: allocator<char>, unknown type: __gnu_cxx::new_allocator<char> 
~~~
### Warnings
~~~
Error in <TStreamerInfo::Build>: allocator<char>, unknown type: __gnu_cxx::new_allocator<char> 
~~~
## std::allocator<double>
### Errors
~~~
Error in <TStreamerInfo::Build>: allocator<double>, unknown type: __gnu_cxx::new_allocator<double> 
~~~
### Warnings
~~~
Error in <TStreamerInfo::Build>: allocator<double>, unknown type: __gnu_cxx::new_allocator<double> 
~~~
## std::allocator<int>
### Errors
~~~
Error in <TStreamerInfo::Build>: allocator<int>, unknown type: __gnu_cxx::new_allocator<int> 
~~~
### Warnings
~~~
Error in <TStreamerInfo::Build>: allocator<int>, unknown type: __gnu_cxx::new_allocator<int> 
~~~
## edm::RefToBase<reco::WMuNuCandidatePtr>
### Errors
~~~
Error in <TStreamerInfo::Build:>: edm::RefToBase<reco::WMuNuCandidatePtr>: edm::reftobase::BaseHolder<reco::WMuNuCandidatePtr>* has no streamer or dictionary, data member holder_ will not be saved
~~~
### Warnings
~~~
Error in <TStreamerInfo::Build:>: edm::RefToBase<reco::WMuNuCandidatePtr>: edm::reftobase::BaseHolder<reco::WMuNuCandidatePtr>* has no streamer or dictionary, data member holder_ will not be saved
~~~
## fakeMenu
### Errors
~~~
Error in <TStreamerInfo::Build>: The class "AlgoMap" is compiled and its base class "map<string,Algo>" is a collection and we do not have a dictionary for it, we will not be able to read or write this base class.
~~~
### Warnings
~~~
Error in <TStreamerInfo::Build>: The class "AlgoMap" is compiled and its base class "map<string,Algo>" is a collection and we do not have a dictionary for it, we will not be able to read or write this base class.
~~~
## pat::DiObjectProxy
### Errors
~~~
Error in <TStreamerInfo::Build:>: pat::DiObjectProxy: type_info* has no streamer or dictionary, data member type1_ will not be saved
Error in <TStreamerInfo::Build:>: pat::DiObjectProxy: type_info* has no streamer or dictionary, data member type2_ will not be saved
~~~
### Warnings
~~~
Error in <TStreamerInfo::Build:>: pat::DiObjectProxy: type_info* has no streamer or dictionary, data member type1_ will not be saved
Error in <TStreamerInfo::Build:>: pat::DiObjectProxy: type_info* has no streamer or dictionary, data member type2_ will not be saved
~~~
[..]
```